### PR TITLE
Introducing Iperf3 sensor

### PR DESCRIPTION
**Pull request by tchellomello**
**Description:**
Introducing Iperf3 client sensor. With this sensor is possible to use [iperf3](http://software.es.net/iperf/) to test your network connection against a local or remote Iperf3 server. 

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#14213

## Checklist:

- [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Comment by syssi** [comment]
LGTM just the config schema should be improved. What about a list of time periods instead of the bunch of config keys (hour, minute, second)?

File: source/_components/sensor.iperf3.markdown [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+description: "How to use Iperf3 within Home Assistant to measure your network bandwith."
+date: 2018-05-01 02:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: iperf3.png
+ha_category: System Monitor
+featured: false
+ha_release: 0.69
```
Comment (syssi) [inline source/_components/sensor.iperf3.markdown]: Please update this to 0.71

File: source/_components/sensor.iperf3.markdown [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  scan_interval:
+    description: Specify the frequency in seconds which the test will be perfomed. Default value is 30 minutes.
+    required: false
+    default: 1800
+    type: int
+{% endconfiguration %}
+
+You can find a list of public Iperf3 servers [here](https://iperf.fr/iperf-servers.php). You can also start your own Iperf3 server using the [mlabbe/iperf3's](https://hub.docker.com/r/mlabbe/iperf3/) docker image or just refer to your `iperf3` command's man page.
+
+You can use the service `sensor.update_iperf3_<server_name>` to run a manual speed test and populate the data or just wait its frequency by setting the value `scan_interval` in seconds.
```
Comment (syssi) [inline source/_components/sensor.iperf3.markdown]: Please update the service name.

**Comment by tchellomello** [comment]
@syssi  documentation updated. Thanks for your code review! :+1:

**Review by syssi — APPROVED** [review]

**Pull request merged**